requirements/requests_awscli.txt

```diff
@@ -1,2 +1,4 @@
--r requests.txt
-awscli==1.29.45
+-c constraints.txt
+
+requests
```

Comment:
```suggestion
-r requests.txt
```
Same as previously, either we authorise mixing `-r` and `-c` (and we can keep several levels of `-r` inclusions) or not. 
I would maybe vote not to have intermediary levels of inclusion, to make the rules simpler

Reply:
Agreed, fixed !